docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
  db:
     image: mcr.microsoft.com/mssql/server:2019-latest
     labels:
-      - webNotes-db
+      - webnotes-db
```

Comment:
Do you have some problems with that? It's no blaming, just question and curiosity :)